src/Microsoft.OData.Client/ALinq/ExpressionWriter.cs

```diff
@@ -506,7 +506,7 @@ internal override Expression VisitConstant(ConstantExpression c)
                 MemberInfo member = typeAnnotation.ElementType.GetField(c.Value.ToString());
                 string memberValue = ClientTypeUtil.GetServerDefinedName(member);
                 ODataEnumValue enumValue = new ODataEnumValue(memberValue, typeNameInEdm ?? typeAnnotation.ElementTypeName);
-                result = ODataUriUtils.ConvertToUriLiteral(enumValue, CommonUtil.ConvertToODataVersion(this.uriVersion), null);
+                result = ODataUriUtils.ConvertToUriLiteral(enumValue, CommonUtil.ConvertToODataVersion(this.uriVersion), null as IEdmModel);

```

Comment:
>null as IEdmModel [](start = 121, length = 17)

if we change the Ieee... as bool? we can get rid of this.

Reply:
Fixed.